README.md

```diff
@@ -81,11 +87,11 @@ To do so, please make a copy of the file `state-backend-template`, name it `stat
 
 ## Configuration
 
-For your configuration, please make a copy of the file `config_template.tfvars`, name it `config.tfvars` and open the file in a text editor. This file contains all variables that are configurable including documentation of the variables. Please adapt the values before you deploy the resources.
+For your configuration, please rename the file `terraform.tfvars.example` to `terraform.tfvars` and open the file in a text editor. This file contains all variables that are configurable including documentation of the variables. Please adapt the values before you deploy the resources.
```

Comment:
Why did you change this logic from "copying" to "rename"? I think if you rename it and make another "git pull" will the file come back?